### DISCUSSION TITLE
"Missing" job schedule definition in 0.8.2

### DISCUSSION THREAD

**Discussion by ctaylor-carvana** [databrickslabs/dbx#550] [Q&A]
Cloning this as a template
https://github.com/databrickslabs/dbx/blob/77cb903d68a35ca3771d8bea8125a6b16199139a/docs/reference/deployment.md#material-code-array-configuring-complex-deployments

New version of dbx complains missing pause status, even though it is clearly included in the example.
workflows -> 6 -> Workflow -> schedule -> pause_status
  field required (type=value_error.missing)
  
The issue was actually incorrectly merged maps in a complex deployment config. I wonder is there a way to make the error message more explicit?

**Comment by renardeinside** [comment]
fixed in #560 , will be released in 0.8.3
